src/GuardClauses/GuardClauseExtensions.cs

```diff
@@ -220,11 +231,12 @@ public static double OutOfRange(this IGuardClause guardClause, double input, str
         /// <param name="parameterName"></param>
         /// <param name="rangeFrom"></param>
         /// <param name="rangeTo"></param>
+        /// <param name="message">Optional. Custom error message</param>
         /// <returns><paramref name="input" /> if the value is not out of range.</returns>
         /// <exception cref="ArgumentOutOfRangeException"></exception>
-        public static float OutOfRange(this IGuardClause guardClause, float input, string parameterName, float rangeFrom, float rangeTo)
+        public static float OutOfRange(this IGuardClause guardClause, float input, string parameterName, float rangeFrom, float rangeTo, string? message = null)
         {
-            return OutOfRange<float>(guardClause, input, parameterName, rangeFrom, rangeTo);
+            return OutOfRange<float>(guardClause, input, parameterName, rangeFrom, rangeTo, message);
```

Comment:
See above. Ok, there are a lot of these so basically apply to all where we're changing which constructor we're calling...